packages/veui/src/components/Select/Select.vue

```diff
@@ -81,6 +83,11 @@ export default {
     searchable: Boolean,
     options: Array,
     multiple: Boolean,
+    showSelectAll: Boolean,
+    composition: {
+      type: Boolean,
+      default: true
+    },
```

Comment:
```suggestion
    composition: Boolean,
```

这里还是默认关了吧，这个 breaking 感觉应该问题不大